core/runtime.rs

```diff
@@ -512,21 +506,20 @@ impl JsRuntime {
     let mut js_runtime = Self {
       v8_isolate: Some(isolate),
       snapshot_options,
+      snapshot_module_load_cb: options.snapshot_module_load_cb.map(Rc::new),
```

Comment:
Why do we need to store this callback here? Is it in preparation for modules support in realms?

Reply:
I guess it's needed to support maybe spawning a realm in the bootstrap code. We'll very likely never do that, but it's also simpler to have this field than to pass it as an argument to `init_extension_js()` so it can be absent on realms.